Gems/Atom/Feature/Common/Assets/seedList.seed

```diff
@@ -336,6 +336,46 @@
+		</Class>
+		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
+			<Class name="AssetId" field="assetId" version="1" type="{652ED536-3402-439B-AEBE-4A5DBC554085}">
+				<Class name="AZ::Uuid" field="guid" value="{A6F9D127-D963-5D78-B19F-A31E73C5C2B5}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
+				<Class name="unsigned int" field="subId" value="0" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			</Class>
+			<Class name="unsigned int" field="platformFlags" value="1" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			<Class name="AZStd::string" field="pathHint" value="passes/diffuseprobegridtemplates.azasset" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
+		</Class>
+		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
+			<Class name="AssetId" field="assetId" version="1" type="{652ED536-3402-439B-AEBE-4A5DBC554085}">
+				<Class name="AZ::Uuid" field="guid" value="{49D6CA83-7AE4-5BF5-9428-636B9EB6DCC7}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
+				<Class name="unsigned int" field="subId" value="0" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			</Class>
+			<Class name="unsigned int" field="platformFlags" value="1" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			<Class name="AZStd::string" field="pathHint" value="shaders/diffuseglobalillumination/diffuseprobegridprepare.azshader" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
+		</Class>
+		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
+			<Class name="AssetId" field="assetId" version="1" type="{652ED536-3402-439B-AEBE-4A5DBC554085}">
+				<Class name="AZ::Uuid" field="guid" value="{D20ADC05-ECF9-5305-93FC-F0A88AB2072D}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
+				<Class name="unsigned int" field="subId" value="0" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			</Class>
+			<Class name="unsigned int" field="platformFlags" value="1" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			<Class name="AZStd::string" field="pathHint" value="shaders/diffuseglobalillumination/diffuseprobegridquery.azshader" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
+		</Class>
+		<Class name="SeedInfo" field="element" version="2" type="{FACC3682-2ACA-4AA4-B85A-07AD276D18A0}">
+			<Class name="AssetId" field="assetId" version="1" type="{652ED536-3402-439B-AEBE-4A5DBC554085}">
+				<Class name="AZ::Uuid" field="guid" value="{B3B664B6-4340-5A35-9A67-DC00A5361D62}" type="{E152C105-A133-4D03-BBF8-3D4B2FBA3E2A}"/>
+				<Class name="unsigned int" field="subId" value="0" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			</Class>
+			<Class name="unsigned int" field="platformFlags" value="1" type="{43DA906B-7DEF-4CA8-9790-854106D3F983}"/>
+			<Class name="AZStd::string" field="pathHint" value="shaders/diffuseglobalillumination/diffuseprobegridvisualizationprepare.azshader" type="{03AAAB3F-5C47-5A66-9EBC-D5FA4DB353C9}"/>
```

Comment:
the title of this shader and the model added make it sound like this is a debug visualization tool that we might not want in release builds. Maybe the code should be instead updated to not need these to exist, to work?

Or am I mis-understanding what these assets are, and they do have a functional use to end-players of distributed games?

Reply:
The visualization passes, shaders, and sphere model are used for development purposes to debug lighting.

Reply:
Is there anything that currently prevents them from trying to load in release builds?

Reply:
I think for now it might make more sense to temporarily add those seeds to the Multiplayer Sample seed list, to encourage the proper fix for these systems here. If we put in this work around, it will mean that we ship a debug feature in release builds longer than necessary.

However, if people disagree with me, I can drop the requested change and instead change to a ship it. I'm just worried that if this goes in, there's no incentive to address the actual underlying problem.

Reply:
IMO, it's better to leave things in a working but non-optimal state vs a broken state that requires anyone creating release builds to go through the same sort of debugging I just did to discover the same problem and solution. Having me institute a workaround in my project _also_ removes the incentive to address the underlying problem, so why not create functioning release builds for _all_ projects by default in the meantime?

That said, if @dmcdiarmid-ly or someone has concrete plans to fix this in the very near-term future, I'm happy to drop this PR.

Reply:
I guess part of it is how harmful is it to ship the debug content in release builds. For teams really trying to optimize build sizes, this is extra content, including an azmodel, and an entire feature they probably don't want in that build.

Most of the cases I've been working on with release builds have been to get release builds as small as possible (mobile games, or games with frequent updates).

I'll drop my "Requested Changes" and change to a ship it for now so this doesn't block other work.

Reply:
I submitted a GHI for cleaning up the release build behavior of Diffuse Probes: https://github.com/o3de/o3de/issues/14463 .

Reply:
Perfect, thanks.